### PR TITLE
Multi select support

### PR DESCRIPTION
**Pull request by jagadeeshym**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -20,6 +20,7 @@
     "react-datepicker": "^1.6.0",
     "react-dropzone": "^5.0.1",
     "react-redux": "^5.0.7",
+    "react-select": "^2.3.0",
```
Comment (ankitcf) [inline package.json]: Please add it to dependencies section in Readme as well with an explanation in comment

File: readme.md [hunk 2]
```diff
@@ -167,6 +167,24 @@ For fields with _type_: "select", another field _foreign_ is mandatory. This fie
 ]
 ```
 
+###MultiSelect Field
```
Comment (ankitcf) [inline readme.md]: Please also add these field type in the "fields" section of readme as well

File: package.json (hunk 1 above)
Comment (jagadeeshym) [reply package.json]: Added

File: readme.md (hunk 2 above)
Comment (jagadeeshym) [reply readme.md]: Added

**Pull request merged**